assignments/01-ownership-borrowing/analysis.md

```diff
@@ -0,0 +1,3 @@
+It's not possible to implement a doubly linked list in safe rust because one object cannot have two owners.
+
+Another example of something that is hard to implement because of Rust's ownership system is UI state. Wh
```

Comment:
Incomplete sentence :standing_person: 
(dw no points removed bc rubric only asked for an example, not an explanation :cartwheeling:)